scripts/dts/gen_dts_cmake.py

```diff
@@ -95,6 +96,7 @@ def main():
         for alias in node.aliases:
             cmake_props.append(f'"DT_ALIAS|{alias}" "{path}"')
 
+    comp_iter = defaultdict(list)
```

Comment:
Naming nitpick here. `iter` is a built-in python function that returns an iterator over an iterable. So the name `comp_iter` suggests it is an iterator. But it's not -- it's a dict, which is not an iterator.

I suggest renaming this `compatible2paths` to match the general edtlib convention of naming a dict `<key>2<value>`

Reply:
yup, done.